lib/ui/profitloss/ProfitLossList.dart

```diff
@@ -0,0 +1,164 @@
+}
+
+abstract class ProfitLossStyle {
+
+  final TextStyle titleTextStyle;
+  final TextStyle titleDetailTextStyle;
+  final TextStyle subTitleTextStyle;
+
+  final Color floatingButtonBackgroundColor;
+  final EdgeInsets generalMargins;
+
+  final double titleDetailEdgePadding;
+
+  final double floatingButtonSize;
+  final double floatingButtonElevation;
+  final double floatingButtonIconSize;
+
+  ProfitLossStyle(this.floatingButtonBackgroundColor, this.floatingButtonSize,
+      this.floatingButtonElevation, this.floatingButtonIconSize, this.generalMargins,
+      this.titleTextStyle, this.titleDetailEdgePadding, this.titleDetailTextStyle,
+      this.subTitleTextStyle);
+}
+
+class _DefaultProfitLossStyle implements ProfitLossStyle{
+
+  final TextStyle titleTextStyle = const TextStyle(fontSize: 47, fontWeight: FontWeight.bold, color: Color(0xFF1a1b46));
+  final TextStyle titleDetailTextStyle = const TextStyle(fontSize: 15, fontWeight: FontWeight.normal, color: Color(0xFF767690));
+  final TextStyle subTitleTextStyle = const TextStyle(fontSize: 17, fontWeight: FontWeight.normal, color: Color(0xFF25df0c));
+
+  final EdgeInsets generalMargins = const EdgeInsets.only(left: 33, top: 36.5, bottom: 12.5);
+
+  final double titleDetailEdgePadding = 10;
+
+  final Color floatingButtonBackgroundColor = const Color(0xFF25df0c);
+  final double floatingButtonSize = 48.0;
+  final double floatingButtonElevation = 0;
+  final double floatingButtonIconSize = 20.0;
+
+  const _DefaultProfitLossStyle();
+}
+
+class ProfitLossPage extends StatefulWidget {
+  State<StatefulWidget> createState() {
+    return _ProfitLossState();
+  }
+}
+
+class _ProfitLossState extends State<ProfitLossPage>  {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: StoreConnector<AppState, MyProfitLossViewModel>(
+          builder: (_, viewModel) => _buildBody(context, viewModel, buildProfitLossViewModel()),
+          converter: (store) => MyProfitLossViewModel.fromStore(store)),
+    );
+  }
+
+  Widget _buildBody(BuildContext context, MyProfitLossViewModel viewModel, ProfitLossViewModel profitLossViewModel, {ProfitLossStyle profitStyle = const _DefaultProfitLossStyle()}) {
+    switch (viewModel.loadingStatus) {
+      case LoadingStatus.LOADING:
+        return Container(
+            child: CircularProgressIndicator(), alignment: Alignment.center);
+      case LoadingStatus.SUCCESS:
+        return _buildPageWithFloatingButton(context, viewModel, profitLossViewModel, profitStyle);
+      case LoadingStatus.ERROR:
+        return Text(Strings.errorString);
+    }
+  }
+
+  Widget _buildPage(BuildContext context, MyProfitLossViewModel viewModel, ProfitLossViewModel profitLossViewModel, ProfitLossStyle profitStyle) {
+    final viewModel = buildProfitLossViewModel();
+    return HeaderAndFooterListView.builder(
+      //FIXME: itemCount is now hardcoded
+        itemCount: 10,
```

Comment:
Use the ViewModel and inject a mock. don't add any test code to the widgets. This is the point of the ViewModel.
We want to be able to sign off the widget in this PR, and make no changes when we hook it up to real data.